### DISCUSSION TITLE
Notes from the Laboratory: July 2023

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#399] [Comments]
# Notes from the Laboratory: July 2023

I am trying this new “thing” of “reporting” what I was up to in a certain period this year, and monthly reviews just sound like something normal people would do. Bear with me while I am typing up this report.

https://kollitsch.dev/blog/2023/notes-from-the-laboratory-july-2023/

**Comment by brycewray** [comment]
Presumably *verschlimmbessern* is a German way of saying that old (?) engineering proverb, “The better is the enemy of the good.”

**Reply by davidsneighbour** [reply]
Yep :D

**Reply by davidsneighbour** [reply]
I just found this here: disimprove, which basically covers it all.

https://en.wiktionary.org/wiki/disimprove#English
